madminer/core/madminer.py

```diff
@@ -795,6 +797,7 @@ def run_multiple(
         initial_command=None,
         python2_override=False,
         systematics=None,
+		order='LO',
     ):
 
         """
```

Comment:
Same here: `order` is not yet documented in the docstring.

Reply:
Also here, some documentation now added in the docstring